rsciio/digitalsurf/_api.py

```diff
@@ -1191,29 +2031,39 @@ def post_process_RGB(signal):
 
         return signal
 
+    @staticmethod
+    def post_process_binary(signal):
+        signal.change_dtype("bool")
+        return signal
```

Comment:
It would be enough. If you think it's better I can directly cast it as a bool array (also probably more instructions-efficient). This was implemented unified behaviour of the cast to RGB, which falls back onto `hyperspy.signal.BaseSignal.change_dtype()`.

Reply:
Okay. If this is possible to specify the type when creating the array, it may be better to do that in order to save on memory usage? If not, this can be kept as it is - slightly less straightforward to read but since this is used in case of RGB type, this is fine.

Reply:
I see. In the .sur file format, bool data are encoded on 16 bits integer. Due to decompression I have to read the full buffer no matter what. The simplest is to initialize the array as int16, and then convert to bool. I don't think doing it immediately or through the hyperspy interface makes much of a difference, what do you think?

Reply:
I guess, this is fine to leave as it is!